launchdarkly-server-sdk/src/data_source.rs

```diff
@@ -52,6 +52,8 @@ pub(crate) struct DeleteData {
     version: u64,
 }
 
+pub type EventProcessedAt = Arc<dyn Fn(std::time::SystemTime) + Send + Sync>;
```

Comment:
Question: does this signature seem reasonable?

Reply:
Seems weird to pass a `SystemTime` here given that its value is always `SystemTime::now()`. I'd understand passing a time if the time were derived from a property on the event, but given that it's just computed at the time the callback is invoked, seems simpler to have the callback itself call `SystemTime::now()`, if it is interested in the system time.

If anything, I think it would make the most sense to pass the [`eventsource_client::SSE`](https://docs.rs/eventsource-client/latest/eventsource_client/enum.SSE.html) event itself to the callback. But we don't need that for our use case, so seems fine for the callback to have signature `fn() -> ()`.